python/python/glide/async_commands/core.py

```diff
@@ -869,6 +869,35 @@ async def scard(self, key: str) -> int:
         """
         return cast(int, await self._execute_command(RequestType.SCard, [key]))
 
+    async def sismember(
+        self,
+        key: str,
+        member: str,
+    ) -> int:
+        """
+        Returns if `member` is a member of the set stored at `key`.
+
+        See https://redis.io/commands/sismember/ for more details.
+
+        Args:
+            key (str): The key of the set.
+            member (str): The member to check for existence in the set.
+
+        Returns:
+            int: 1 if the member exists in the set, 0 otherwise.
+            If `key` is not a set, it is treated an an empty set and the command returns 0.
```

Comment:
If `key` is not a set, it is treated an an empty => If `key` is not a set, it is treated as an empty s